tests/api/themes.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * Themes REST API Test
+ *
+ * @package WooCommerce Admin\Tests\API
+ */
+
+/**
+ * WC Tests API Themes
+ */
+class WC_Tests_API_Themes extends WC_REST_Unit_Test_Case {
+
+	/**
+	 * Endpoints.
+	 *
+	 * @var string
+	 */
+	protected $endpoint = '/wc-admin/v1/themes';
+
+	/**
+	 * Setup test data. Called before every test.
+	 */
+	public function setUp() {
+		parent::setUp();
+
+		$this->user = $this->factory->user->create(
+			array(
+				'role' => 'administrator',
+			)
+		);
+	}
+
+	/**
+	 * Test that an error is thrown if the file is missing.
+	 */
+	public function test_missing_upload() {
```

Comment:
Might be nice to add a test case for a user without the proper role attempting to upload a theme.

Reply:
Added in https://github.com/woocommerce/woocommerce-admin/pull/2568/commits/36f9325228a0041f5c1e00fdc3743659b3992582